projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/notebook.py

```diff
@@ -46,28 +47,47 @@ class Notebook:
     Other cells are ignored.
     """
 
-    def __init__(self, file_path: Path):
+    @staticmethod
+    def register_notebook_steps(file_path: Path, context: JobContext):
         try:
-            self.sql_and_run_cells = []
-            self.python_helper_cells = []
-            self.file_path = file_path
+            python_module = imp.new_module("notebook")
+            exec("job_input = 1", python_module.__dict__)
```

Comment:
What is this?

Reply:
I do it in order to pass our  job_input variable in the module. I declare it like that then access it via module.job_input = job_input (the second one is ours). I couldn't find another way to pass the "real" job_input, so I use it as that. Open to recommendation on using something else to pass job_input into the module

Reply:
Ok. Currently I am not sure. I feel there should be. 

Do add a comment above this snippet of code that explains the reason . 
It's a good idea any place that the reason for doing something is not obvious from the code itself to comment it.